src/serverEmulator.ts

```diff
@@ -173,9 +172,12 @@ export default class ServerEmulator {
 
     private resetEncoder(): GIFEncoder {
         const encoder = new GIFEncoder(this.width, this.height)
+        
+        const repeat : number = settingsMap.get(this.message?.channel.id ?? "", 'loop_gifs') == 'true' ? 0 : -1
```

Comment:
The same thing with passing an empty string instead of passing null, also true should be boolean IMO.